src/messages/manifestjson.js

```diff
@@ -676,3 +717,53 @@ export const EXTENSION_ID_REQUIRED = {
   ),
   file: MANIFEST_JSON,
 };
+
+export const PRIVILEGED_FEATURES_REQUIRED = 'PRIVILEGED_FEATURES_REQUIRED';
+export function privilegedFeaturesRequired(error) {
+  const messageTmpl = i18n._(
+    oneLine`%(instancePath)s: Privileged extensions should declare privileged permissions.`
+  );
+
+  const message = i18n.sprintf(messageTmpl, {
+    instancePath: error.instancePath,
+  });
+
+  return {
+    code: PRIVILEGED_FEATURES_REQUIRED,
+    message,
+    description: i18n._(oneLine`
+      This extension does not declare any privileged permission. It does not need to be signed with the privileged certificate.
+      Please upload it directly to "https://addons.mozilla.org/".
```

Comment:
Do we need the quotes around `https://addons.mozilla.org/`?

Reply:
Not necessarily, I always quote urls to make it clear where they start and end, and so I have done it without realizing it wasn't quoted in your suggested version.

I removed the quote, given that the url is known (and not an arbitrary one interpolated into the message) I don't think we really need to quote it.